src/bloom.cpp

```diff
@@ -112,14 +112,7 @@ bool CBloomFilter::CheckScript(const CScript &script) const
 
 bool CBloomFilter::CheckPayeeSharesScripts(const std::vector<PayoutShare>& payoutShares) const
```

Comment:
nit: maybe better name is `cointainsPayeeShareScript` or something similar? `check` makes me confused like that's kind of validation, but ignore me if I am wrong so far as English is not my native lang